app/.env.example

```diff
@@ -19,5 +19,5 @@ ENABLE_MOCK_TIME=<true or false>
 ENABLE_MOCK_AUTH=<true or false>
 
 # These variables are for setting the CIF program director name and email
-NEXT_PUBLIC_PROGRAM_DIRECTOR_NAME=
-NEXT_PUBLIC_PROGRAM_DIRECTOR_EMAIL=
```

Comment:
They needed prefixes when being used via `process.env`, but since it's convict now, I shortened the names for clarity